### PR TITLE
if resizeObserver does not find a parent el it checks for SDOM host

### PR DESCRIPTION
**Pull request by dkrasner**
### Main Points ###

Takes care of issue #11

### CODE REVIEW

File: src/GridSheet.js [hunk 1]
```diff
@@ -196,7 +196,13 @@ class GridSheet extends HTMLElement {
             this.setAttribute('tabindex', '-1');
 
             // Set up the resizing observer
-            this.observer.observe(this.parentElement);
+            let parentElement = this.parentElement;
+            if(!parentElement){
+                // if the DOM parent is not present see if GridSheet is attached
+                // to a shadow DOM host
+                parentElement = this.shadowRoot.host;
```
[the remark this answers is not in the document]
Comment (dkrasner) [reply src/GridSheet.js:203]: ah yeah that is my mistakes in switching between the `Worksheet` and the `GridSheet` components... 


however that still doesn't work right... the quick solution is just to append `GridSheet` as a child of something else in the SDOM, but there should be a way to make this work 

l'll look into it and ping

**Comment by dkrasner** [comment]
@darth-cheney take a look, this seems to work ok (the resizing is a bit jittery when the sheet is a child in the SDOM of some host)

Note. [this](https://github.com/darth-cheney/ap-sheet/pull/12/commits/be68aa2e2f87352d484f60b3f683ecbed421a657#diff-ad6cea5532b3c5febcaf872e88b3c81b25cf47e0ac8d32b91590eeab616b0cf6R289) change. Before there was no reference to the _actual_ target element of the ResizeObserver, and I think this was just working b/c the target and ap-sheet matched, which won't always be the case (as in the SDOM example)

**Comment by darth-cheney** [comment]
Good catch. This works on my end. Should we take this opportunity to add the ability to expand vertically and also to set the three corresponding element attributes (vertical, horizontal, both) etc?

**Comment by dkrasner** [comment]
lets merge, since i am working primarily off this branch and I (or you) can take care of the row/both next. Can do it as soon as I get this Worksheet comp PR wrapped..

**Pull request merged**